code/addons/docs/src/types.ts

```diff
@@ -0,0 +1,223 @@
+import type { ModuleExport, ModuleExports } from '@storybook/types';
+
+type StoryBlockParameters = {
+  /** Whether a story's play function runs when shown in docs page */
+  autoplay?: boolean;
+  /**
+   * Set a minimum height (note for an iframe this is the actual height) when rendering a story in
+   * an iframe or inline. This overrides parameters.docs.story.iframeHeight for iframes.
+   */
+  height?: string;
+  /** IFrame configuration */
+  iframeHeight?: string;
+  /**
+   * Whether the story is rendered inline (in the same browser frame as the other docs content) or
+   * in an iframe
+   */
+  inline?: boolean;
+  /** Specifies the CSF file to which the story is associated */
+  meta: ModuleExports;
+  /**
+   * Specifies which story is rendered by the Story block. If no of is defined and the MDX file is
+   * attached, the primary (first) story will be rendered.
+   */
+  of: ModuleExport;
+};
```

Comment:
I find it helpful to render "code" in backticks:

```suggestion
type StoryBlockParameters = {
  /** Whether a story's play function runs when shown in docs page */
  autoplay?: boolean;
  /**
   * Set a minimum height (note for an iframe this is the actual height) when rendering a story in
   * an iframe or inline. This overrides `parameters.docs.story.iframeHeight` for iframes.
   */
  height?: string;
  /** IFrame configuration */
  iframeHeight?: string;
  /**
   * Whether the story is rendered inline (in the same browser frame as the other docs content) or
   * in an iframe
   */
  inline?: boolean;
  /** Specifies the CSF file to which the story is associated */
  meta: ModuleExports;
  /**
   * Specifies which story is rendered by the Story block. If no `of` is defined and the MDX file is
   * attached, the primary (first) story will be rendered.
   */
  of: ModuleExport;
};
```